page-objects/pages/TASKPage.js

```diff
@@ -169,6 +170,10 @@ async function clickCompletedTasksTab(){
 	await elementHelpers.clickAndWait(completedTasksTab);
 }
 
+async function clickFinishedTasksTab(){
```

Comment:
clickOpenTasksTab, clickCompletedTasksTab, clickFinishedTasksTab and click clickDraftTasksTab are also candidates to have it in one function instead of several ones. Similar to function getTaskActionBtnSelector in TASKListPage.js.